src/main/java/ch/njol/skript/events/EvtItem.java

```diff
@@ -61,10 +62,17 @@ public class EvtItem extends SkriptEvent {
 				.examples("on item spawn of iron sword:",
 						"\tbroadcast \"Someone dropped an iron sword!\"")
 				.since("<i>unknown</i> (before 2.1)");
-		Skript.registerEvent("Drop", EvtItem.class, PlayerDropItemEvent.class, "[player] drop[ing] [[of] %-itemtypes%]")
-				.description("Called when a player drops an item from their inventory.")
-				.examples("on drop:")
-				.since("<i>unknown</i> (before 2.1)");
+		Skript.registerEvent("Drop", EvtItem.class, CollectionUtils.array(PlayerDropItemEvent.class, EntityDropItemEvent.class),
+				"[player|1:entity] drop[ing] [[of] %-itemtypes%]")
```

Comment:
```suggestion
				"[player|1:entity] drop[ping] [[of] %-itemtypes%]")
```
The gerund of "drop" is "dropping" but that pattern allows for "droping" which is not correct

Reply:
jebus, how did I screw that one up? haha, oops! thanks for that!